sklego/pandas_utils.py

```diff
@@ -86,8 +89,10 @@ def wrapper(*args, **kwargs):
         func_args_str = "".join(
```

Comment:
`inspect.signature` only needs to happen if we print the args, so it's probably better to move it into the `if` statement below